lib/colorprompt.g

```diff
@@ -13,6 +13,9 @@
 # see comment below
 if not IsBound(STDOut) then
   BindGlobal("STDOut", OutputTextUser());
+  if IsBound(HPCGAP) then
+    MakeReadOnly(STDOut);
+  fi;
```

Comment:
I went and fixed this properly instead, it just needed some lists making immutable.